### PR TITLE
RUN-2032: Percentage in number showing in progress bar instead of undefined%

### PR DESCRIPTION
**Pull request by jayas006**
**Is this a bugfix, or an enhancement? Please describe.**

"undefined%" message appears in the progress bar instead of percentage. Fixed it to make it shows the percentage in number. 

**Describe the solution you've implemented**

**Describe alternatives you've considered**

**Additional context**

### CODE REVIEW

File: rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue [hunk 1]
```diff
@@ -288,29 +288,31 @@
...
-                :value="100"
+                :modelValue="100"
                 striped
                 type="default"
                 label
                 :label-text="
                   $t('job.execution.starting.0', [
-                    runningStartedDisplay(exec.dateStarted.date),
+                    // if exec.dateStarted is null or undefined,it doesn't throw an error but instead returns undefined
+                    runningStartedDisplay(exec.dateStarted?.date) || 'N/A',
```
Comment (smartinellibenedetti) [inline rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue:301]: Question, what led you to the conclusion that this change was needed? Have you seen a scenario where this happens?

File: rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue [hunk 2]
```diff
@@ -801,6 +802,25 @@ export default defineComponent({
...
+      if (
+        exec.job &&
+        exec.job.averageDuration &&
+        exec.dateStarted &&
+        exec.dateStarted.date
+      ) {
+        const now = moment();
+        const startDate = moment(exec.dateStarted.date);
+        const diff = now.diff(startDate);
+        const percentage = Math.floor((diff / exec.job.averageDuration) * 100);
```
Comment (smartinellibenedetti) [inline rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue:817]: One heads-up, if it's the first time running a job, `averageDuration` isn't available. We shouldn't be showing 0 in this case.

File: rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue [hunk 3]
```diff
@@ -801,6 +802,25 @@ export default defineComponent({
       }
       return 0;
     },
+
+    // Below code makes the progress bar reach closer to 100%
```
Comment (smartinellibenedetti) [inline rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue:806]: ✂️

**Review by smartinellibenedetti — COMMENTED** [review]
One heads-up, one common habit that engineers develop is to validate solutions before implementing them to avoid missing important aspects.

The architecture meeting on Mondays is an example of folks validating their ideas.

Whenever starting to work on a task that doesn't have steps regarding how to solve it, l'd like you to investigate the issue and send me or other folks a message with the plan of how this issue will be solved.

In this type of communication, it is also helpful to point out how you concluded that what you're solving is the problem in the first place.

This will help us to ensure you're on the right track, and that you have all the info you need, so that once you execute your plan, the PR is ready for review.

Currently, this will be moved back to in progress, and we can discuss it more tomorrow.

File: rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue (hunk 1 above)
Comment (jayas006) [reply rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue:301]: > Question, what led you to the conclusion that this change was needed? Have you seen a scenario where this happens?

I was going through line by line code between progress bar (first I saw in vue dev tool to confirm which line of code I have to see to make it  right) and started understanding the functions. So the `runningStartedDispaly` requires the valid date to return the right output.

Comment (jayas006) [reply rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue]: > Question, what led you to the conclusion that this change was needed? Have you seen a scenario where this happens?

Ok so I fixed it now. It was just unnecessary change from my side. Ignore it

**Review by smartinellibenedetti — APPROVED** [review]
good job!

**Pull request merged**